nose_parameterized/parameterized.py

```diff
@@ -228,7 +270,15 @@ def parameterized_expand_wrapper(f):
                     if len(p.args) > 0 and isinstance(p.args[0], compat.string_types):
                         name_suffix += "_" + cls.to_safe_name(p.args[0])
                     name = base_name + name_suffix
+
+                if testcase_func_doc:
```

Comment:
Please refactor this so that what is now the `doc_expansion` function is defined at the module level, called something like `default_testcase_func_doc`, and has the same signature as the user-supplied testcase doc func would have (ie, so this if-statement is removed and there's just one line which is `testcase_func_doc = testcase_func_doc or default_testcase_func_doc`.